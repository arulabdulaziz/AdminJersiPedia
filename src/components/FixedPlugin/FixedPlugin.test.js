import TestRenderer from "react-test-renderer";
import FixedPlugin from "./FixedPlugin";

describe("FixedPlugin", () => {
  describe("Using react-test-renderer", () => {
    it("using props 1", () => {
      const props = {
        bgColor: "white",
        handleBgClick: jest.fn(),
        activeColor: "primary",
        handleActiveClick: jest.fn(),
      };
      const tree = TestRenderer.create(<FixedPlugin {...props} />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("using props 2", () => {
        const props = {
          bgColor: "black",
          handleBgClick: jest.fn(),
          activeColor: "secondary",
          handleActiveClick: jest.fn(),
        };
        const tree = TestRenderer.create(<FixedPlugin {...props} />).toJSON();
        expect(tree).toMatchSnapshot();
      });
  });
});
