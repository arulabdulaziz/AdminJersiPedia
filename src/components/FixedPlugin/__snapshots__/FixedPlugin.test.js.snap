// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FixedPlugin Using react-test-renderer using props 1 1`] = `
<div
  className="fixed-plugin"
>
  <div
    className="dropdown show"
  >
    <div
      onClick={[Function]}
    >
      <i
        className="fa fa-cog fa-2x"
      />
    </div>
    <ul
      className="dropdown-menu show"
    >
      <li
        className="header-title"
      >
        SIDEBAR BACKGROUND
      </li>
      <li
        className="adjustments-line"
      >
        <div
          className="badge-colors text-center"
        >
          <span
            className="badge filter badge-dark"
            data-color="black"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-light active"
            data-color="white"
            onClick={[Function]}
          />
        </div>
      </li>
      <li
        className="header-title"
      >
        SIDEBAR ACTIVE COLOR
      </li>
      <li
        className="adjustments-line"
      >
        <div
          className="badge-colors text-center"
        >
          <span
            className="badge filter badge-primary active"
            data-color="primary"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-info"
            data-color="info"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-success"
            data-color="success"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-warning"
            data-color="warning"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-danger"
            data-color="danger"
            onClick={[Function]}
          />
        </div>
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-primary btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-react?ref=pdr-fixed-plugin"
          onClick={[Function]}
        >
          Download now
        </a>
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-outline-default btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-react/#/documentation/tutorial?ref=pdr-fixed-plugin"
          onClick={[Function]}
        >
          <i
            className="nc-icon nc-paper"
          />
           Documentation
        </a>
      </li>
      <li
        className="header-title"
      >
        Want more components?
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-danger btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-pro-react?ref=pdr-fixed-plugin"
          onClick={[Function]}
          target="_blank"
        >
          Get pro version
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`FixedPlugin Using react-test-renderer using props 2 1`] = `
<div
  className="fixed-plugin"
>
  <div
    className="dropdown show"
  >
    <div
      onClick={[Function]}
    >
      <i
        className="fa fa-cog fa-2x"
      />
    </div>
    <ul
      className="dropdown-menu show"
    >
      <li
        className="header-title"
      >
        SIDEBAR BACKGROUND
      </li>
      <li
        className="adjustments-line"
      >
        <div
          className="badge-colors text-center"
        >
          <span
            className="badge filter badge-dark active"
            data-color="black"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-light"
            data-color="white"
            onClick={[Function]}
          />
        </div>
      </li>
      <li
        className="header-title"
      >
        SIDEBAR ACTIVE COLOR
      </li>
      <li
        className="adjustments-line"
      >
        <div
          className="badge-colors text-center"
        >
          <span
            className="badge filter badge-primary"
            data-color="primary"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-info"
            data-color="info"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-success"
            data-color="success"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-warning"
            data-color="warning"
            onClick={[Function]}
          />
          <span
            className="badge filter badge-danger"
            data-color="danger"
            onClick={[Function]}
          />
        </div>
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-primary btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-react?ref=pdr-fixed-plugin"
          onClick={[Function]}
        >
          Download now
        </a>
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-outline-default btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-react/#/documentation/tutorial?ref=pdr-fixed-plugin"
          onClick={[Function]}
        >
          <i
            className="nc-icon nc-paper"
          />
           Documentation
        </a>
      </li>
      <li
        className="header-title"
      >
        Want more components?
      </li>
      <li
        className="button-container"
      >
        <a
          aria-label={null}
          className="btn-round btn btn-danger btn-block"
          href="https://www.creative-tim.com/product/paper-dashboard-pro-react?ref=pdr-fixed-plugin"
          onClick={[Function]}
          target="_blank"
        >
          Get pro version
        </a>
      </li>
    </ul>
  </div>
</div>
`;
